tasks/maintainers.rb

```diff
@@ -0,0 +1,67 @@
+#
+# Author:: Adam Jacob (<adam@opscode.com>)
+# Author:: Daniel DeLeo (<dan@opscode.com>)
+# Copyright:: Copyright (c) 2008, 2010 Opscode, Inc.
```

Comment:
Mind fixing this up?  Should be Chef Software, Inc.  And we can probably just leave the Author:: lines out.